app/layout/nav_entries.phtml

```diff
@@ -1,5 +1,7 @@
-<ul id="nav_entries">
-	<li class="item"><a class="previous_entry" href="#"><?php echo _i('prev'); ?></a></li>
-	<li class="item"><a class="up" href="#"><?php echo _i('up'); ?></a></li>
-	<li class="item"><a class="next_entry" href="#"><?php echo _i('next'); ?></a></li>
-</ul>
\ No newline at end of file
+<div <?php if (FreshRSS_Context::$user_conf->hide_nav_entries) echo 'hidden'; ?> > <!-- adding hidden to the ul doesn't have the desired effect -->
```

Comment:
It would be better to not render that template at all instead of rendering it and hidding it.
You could move the test in ```app/views/index/normal.phtml```.

Reply:
init_nav_entries() in p/scripts/main.js looks for it by id, might need to test to make sure that doesn't cause problems if it's not rendered at all.

Reply:
If there is a problem in js when the tag is not rendered, you should fix the js instead of just hiding the content.

Reply:
I tested it, and init_nav_entries() doesn't have a problem, but it looks like the [StickyFeeds Extension](https://github.com/FreshRSS/Extensions/blob/b4aeeefc3fa98ccad0837f329ecd99287fc7a4ae/xExtension-StickyFeeds/static/script.js) may have an issue. 

`$(window).height() - $('#nav_entries').height()` evaluates to `NaN` when `#nav_entries` is not rendered because the height will be undefined. When `#nav_entries` is hidden, `$('#nav_entries').height()` evaluates to `0`.

I've pushed the change here to conditionally render `#nav_entries`, but it may be simpler and less disruptive to revert to hiding it.

Reply:
Good. In that case, we need to fix StickyFeeds :)